js/tests/helpers/stringHelpers.test.js

```diff
@@ -1,24 +1,14 @@
 import {
-	unescapeString,
+	firstToUpperCase,
 } from "../../src/helpers/stringHelpers";
 
-describe( "unescapeString", () => {
-	it( "decodes &#39; from the string to an apostrophe.", () => {
-		const escapedString = "I have apostrophe&#39;s!";
+describe( "firstToUpperCase", () => {
+	it( "sets the first character of a string to uppercase", () => {
+		const escapedString = "i should know better and be capitalized";
```

Comment:
I should start with a capital

Reply:
Yes, that was my point. So it's the string that should have its first letter capitalized by the function we are testing, `firstToUpperCase()`. You can see that the expected result does have the `i` capitalized.